build.assets/Dockerfile-node

```diff
@@ -41,12 +45,21 @@ RUN apt-get -y update && \
     # Used during tag builds to build the RPM package of Connect.
     rpm \
     && \
+    # Manually install the wasm-opt binary on ARM64.
```

Comment:
this comment does not match the line below - it installs binaryen not wasm-opt. Perhaps say "Manually install the wasm-opt binary from the binaryen package on ARM64" just for clarity. To me, this looks like an old comment someone forgot to update :)

Reply:
f2993a1fb606bf9de154b49e93f98dd4513a11c1